src/lib/viewers/BaseViewer.js

```diff
@@ -1294,7 +1295,8 @@ class BaseViewer extends EventEmitter {
             whoHighlighted: __('annotation_who_highlighted'),
         };
         return cloneDeep({
-            ...this.options,
+            ...restOptions,
+            features,
```

Comment:
This seems a little awkward. What is the reason for having it between the two spread operations?

Reply:
The reason is I wanted to make the options object flatter -- without the destructuring of `features` the annotator would get an options object such as:
```
{
  annotationsOptions: {
    features: {
       ...
    },
    ...
}
```